testing/snapshot.go

```diff
@@ -102,6 +122,9 @@ func GenerateSnapshot(t *testing.T, repo *repository.Repository, files []MockFil
 		f(o)
 	}
 
+	tmpBackupDir := GenerateFiles(t, files)
+	repo.AppContext().CWD = tmpBackupDir
```

Comment:
i get the point of GenerateFiles, you need it for ptar, and that's ok for now*, but why plugging it in here as well?  We're not going to look at the `tmpBackupDir` anyway since we use the MockImporter and not the fs one.  Am I missing something?

Reply:
it's not needed, it was an error after rebasing with main, I removed it in https://github.com/PlakarKorp/plakar/pull/859/commits/af072c630b6580811ae3926614596134d120c9c7

Reply:
thank you!